torch/_inductor/runtime/triton_bundler.py

```diff
@@ -0,0 +1,127 @@
+import dataclasses
+import logging
+import os
+from pathlib import Path
+from typing import Dict, List, Optional
+
+import torch
```

Comment:
@jansel When do I put a file in torch._inductor.runtime as opposed to somewhere else?

Reply:
Our generated code should only import from torch._inductor.runtime.